ngraph/frontend/tensorflow/src/op/identity.cpp

```diff
@@ -13,11 +13,14 @@ namespace frontend {
 namespace tf {
 namespace op {
 
-OutputVector TranslateIdentityOp(const NodeContext& node) {
-    return {node.get_ng_input(0)};
+OutputVector translate_identity_op(const NodeContext& node) {
+    auto input = node.get_input(0);
+    set_out_name(node.get_name(), input);
```

Comment:
please make a comment about why we specify several additional names

Reply:
propose to fix it in the next PR

Reply:
fixed in https://github.com/openvinotoolkit/openvino/pull/8477